squire_sdk/src/server/gathering/mod.rs

```diff
@@ -1,15 +1,16 @@
-use std::collections::HashMap;
+use std::{collections::HashMap, ops::Add};
```

Comment:
Why was `std::ops::Add` imported?

Reply:
My ide recommended importing `Add`, because calculating the next check time requires adding a `Duration` to an `Instant`.

```rust
fn get_last_message_checked_at(&self) -> Instant {
    self.last_message_processed_at
        .add(Duration::from_secs(24 * 60 * 60))
}
```

Reply:
Ah, gotcha. You should be able to avoid that in the future by just using the `+` operator.
```rust
fn get_last_message_checked_at(&self) -> Instant {
    self.last_message_processed_at + Duration::from_secs(24 * 60 * 60)
}
```